packages/fuselage/src/components/Modal/Modal.styles.scss

```diff
@@ -48,6 +48,7 @@ $modal-margin: theme('modal-margin', auto);
   &__header-inner {
     display: flex;
     flex-wrap: nowrap;
+    align-items: center;
```

Comment:
Unfortunately, if we use this approach it's going to break the Modal with `Tagline` and `Icon`